RecoParticleFlow/PFProducer/python/particleFlowEGamma_cff.py

```diff
@@ -53,10 +53,30 @@
 egmElectronIsolationPileUpCITK.srcToIsolate = cms.InputTag("gedGsfElectronsTmp")
 egmElectronIsolationPileUpCITK.srcForIsolationCone = cms.InputTag("pfPileUpAllChargedParticles")
 
+photonIDValueMapProducerRECO = cms.EDProducer(
+  "PhotonIDValueMapProducer",
+  ebReducedRecHitCollection = cms.InputTag("ecalRecHit","EcalRecHitsEB"),
+  ebReducedRecHitCollectionMiniAOD = cms.InputTag(""),
+  eeReducedRecHitCollection = cms.InputTag("ecalRecHit","EcalRecHitsEE"),
+  eeReducedRecHitCollectionMiniAOD = cms.InputTag(""),
+  esReducedRecHitCollection  = cms.InputTag("ecalPreshowerRecHit","EcalRecHitsES"),
+  esReducedRecHitCollectionMiniAOD = cms.InputTag(""),
+  particleBasedIsolation = cms.InputTag("particleBasedIsolationTmp","gedPhotonsTmp"),
+  pfCandidates = cms.InputTag("particleFlowTmp"),
+  pfCandidatesMiniAOD = cms.InputTag(""),
+  src = cms.InputTag("gedPhotonsTmp"),
+  srcMiniAOD = cms.InputTag(""),
+  vertices = cms.InputTag("offlinePrimaryVertices"),
+  verticesMiniAOD = cms.InputTag("")
+  )
+
+
 particleFlowEGammaFull = cms.Sequence(particleFlowEGamma*gedGsfElectronSequenceTmp*gedPhotonSequenceTmp*ootPhotonSequence)
 particleFlowEGammaFinal = cms.Sequence(particleBasedIsolationTmp*\
 pfNoPileUpIsoSequence*cms.ignore(pfNoPileUpCandidates)*cms.ignore(pfPileUpAllChargedParticles)*\
-egmPhotonIsolationCITK*egmElectronIsolationCITK*egmElectronIsolationPileUpCITK*gedPhotonSequence*gedElectronPFIsoSequence)
+egmPhotonIsolationCITK*egmElectronIsolationCITK*egmElectronIsolationPileUpCITK*\
+photonIDValueMapProducerRECO*\
```

Comment:
I think that the endline escapes are not needed

Reply:
indeed, they are not, removed